src/Player.ts

```diff
@@ -306,8 +327,16 @@ export class Player {
       // Update the cursors and listeners.
       const entry =
         this._options.converter.timemap[
-          index >= 0 ? index : Math.max(0, -index - 2)
+        index >= 0 ? index : Math.max(0, -index - 2)
         ];
+
+      // Guard against undefined entry
```

Comment:
This is not part of the feature. 

Also, are there concrete cases where this would fail?